subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -1,1006 +0,0 @@
-// Copyright 2017 the original author or authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-[[dependency_management]]
```

Comment:
There are surely a lot of links to dependency-management.html. We must decide where we should redirect this URL to if we're not going to keep *some* page at that location.

Reply:
I guess redirecting to the "Introduction to Dependency Management" would be good enough. Which links are you referring to specifically? You mean in external blog post, guides etc?